custom_config.py

```diff
@@ -45,3 +48,25 @@ def get_whitelisted_ports_from_test_id(self, test_id):
                 return set(rule['ports'])
 
         return set([])
+
+    def considered_inactive(self):
+        considered_inactive = self._parse_user_is_inactive_relative_time('considered_inactive')
+        if considered_inactive is None:
+            return datetime.now(timezone.utc)-relativedelta(years=+1)
```

Comment:
I think there's a UTC now method.

Reply:
[`datetime.utcnow()`](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) is a little different, as it's not timezone-aware. Need that since all of the times from `get_credential_report()` are timezone-aware.